Userland/Applications/Help/CMakeLists.txt

```diff
@@ -13,9 +13,8 @@ set(SOURCES
     main.cpp
     MainWidget.cpp
     ManualModel.cpp
-    ManualPageNode.cpp
-    ManualSectionNode.cpp
 )
 
 serenity_app(Help ICON app-help)
 target_link_libraries(Help LibWebView LibWeb LibMarkdown LibGUI LibDesktop LibMain)
+link_with_unicode_data(Help)
```

Comment:
Looks like this helper changed names between rebases, it's now `link_with_locale_data`